src/utils/transaction.ts

```diff
@@ -1389,9 +1797,9 @@ export const submitEtherspotAndWaitForTransactionHash = async (
     }
 
     const feeToken = isZeroAddress(feeTokenAddress) ? undefined : feeTokenAddress;
-    await sdk.estimateGatewayBatch({ feeToken });
+    await sdk.estimateGatewayBatch({ feeToken: feeToken });
```

Comment:
I think this was fine as before since the var name is the same as the key name (doesn't really matter either way